matter_server/server/device_controller.py

```diff
@@ -1500,3 +1526,55 @@ def _get_node_lock(self, node_id: int) -> asyncio.Lock:
         if node_id not in self._node_lock:
             self._node_lock[node_id] = asyncio.Lock()
         return self._node_lock[node_id]
+
+    async def _custom_attributes_poller(self) -> None:
+        """Poll custom clusters/attributes for changes."""
+        for node_id, polled_attributes in self._polled_attributes.items():
+            node = self._nodes[node_id]
+            if not node.available:
+                continue
+            for attribute_path in polled_attributes:
+                try:
+                    # try to read the attribute(s) - this will fire an event if the value changed
+                    async with self._get_node_lock(node_id):
+                        await self.read_attribute(
+                            node_id, attribute_path, fabric_filtered=False
+                        )
+                except (ChipStackError, NodeNotReady) as err:
+                    LOGGER.warning(
+                        "Polling custom attribute %s for node %s failed: %s",
+                        attribute_path,
+                        node_id,
+                        str(err) or err.__class__.__name__,
+                        # log full stack trace if verbose logging is enabled
+                        exc_info=err
+                        if LOGGER.isEnabledFor(VERBOSE_LOG_LEVEL)
+                        else None,
+                    )
+                # polling attributes is heavy on network traffic, so we throttle it a bit
+                await asyncio.sleep(5)
```

Comment:
This seems rather conservative to me, given that we poll 3 or 4 almost concurrently currently. Maybe 2s?

Reply:
Yeah, I wasn't sure about good/sane values. This is maybe a step too far :-)
I'll change it to 2s indeed.